### ISSUE TITLE
Does using "bazel test" work?

### ISSUE THREAD

**Issue by spyle2** [emscripten-core/emscripten#14998]
I cant seem to use bazel to run a test that includes "emscripten.h" or any other emscripten libs.
I wrote a cc_test and use --config=wasm and have gone through many, many errors that I am slowly working through. cc_binary/cc_libary work fine using "bazel build --config=wasm".

I have gone through many errors; so manual editing of the toolchain flags fixed most. Currently I am stuck on it not being able to include node_modules. 
```
internal/modules/cjs/loader.js:797
    throw err;
    ^

Error: Cannot find module './lib/utils'
Require stack:
```

That specific error isn't really my main question. I really want to know if anyone has gotten cc_test to work at all with including emscripten libs?
Should we create a wasm_cc_test?

**Comment by kripken** [comment]
Looks like a duplicate of https://github.com/emscripten-core/emsdk/issues/883 ?

Closing in favor of that. In general, please don't open duplicate issues unless there is a specific reason for doing so - the conversation can get split up, and also it just pings the same people twice for the same issue.
